av_metrics/c-example/test.c

```diff
@@ -6,20 +6,20 @@
     void test_ ## func(int frame) { \
         const char video_path1[] = "../testfiles/yuv444p8_input.y4m"; \
         const char video_path2[] = "../testfiles/yuv444p8_output.y4m"; \
-        const MetricContext *val = calculate_ ## func(&video_path1[0], \
+        const AVMMetricContext *val = avm_calculate_ ## func(&video_path1[0], \
```

Comment:
AVMMetricContext is a typo magnet. AVMContext maybe?

Reply:
Yeah I agree, that is why I was hesitant to use `AVM` as the prefix, but `AV` would have made it overlap with the `libav*` libraries. I can change it to just be `Context` in `capi.rs` (resulting in `AVMContext`) if you like.

Reply:
Sounds good to me.